src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -166,6 +166,20 @@ case class AttributeValueList(val list: Seq[AttributeValue]) extends AttributeVa
 case class AttributeReferenceList(val list: Seq[AttributeReferenceSingle]) extends AttributeReference // non-recursive
 case class AttributeReferenceSingle(val entityType: String, val entityName: String) extends AttributeReference
 
+@ApiModel(value = "JobDescription")
+case class JobDescription(
+  @(ApiModelProperty@field)(required = true, value = "The method to execute.")
+  methodName: MethodConfigurationName,
+  @(ApiModelProperty@field)(required = true, value = "The entity that supplies parameters to the method.")
+  entityName: String)
```

Comment:
I understand that the entity type is in the method config but I feel like not including it here will lead to poor error messaging. E.g. I am submitting an individual but the method config says sample, the only error message can be that a sample does not exist with the given name.